### PR TITLE
Templatized Name

### LINKED ISSUE #8: Parameterize project name
`ce3.g8` isn't a sane default for generated projects.

### PR DESCRIPTION
**Pull request by TonioGela**
As per title, this should fix #8 
Worth saying that It templatizes the main filename too.

### CODE REVIEW

File: src/main/g8/src/main/scala/$package$/$name__Camel$.scala [hunk 1]
```diff
@@ -3,7 +3,7 @@ package $package$
 import cats.effect.IOApp
 import cats.effect.IO
 
-object Main extends IOApp.Simple {
+object $name;format="Camel"$ extends IOApp.Simple {
```
Comment (kubukoz) [inline src/main/g8/src/main/scala/$package$/$name__Camel$.scala:6]: I wouldn't change this file. Personally I use this pattern with `Main` as my main class 100% of the time (and I've seen that pattern in prod in both my recent jobs). The package name is enough to disambiguate.

File: src/main/g8/build.sbt [hunk 2]
```diff
@@ -2,7 +2,7 @@ ThisBuild / organization := "com.example"
 ThisBuild / scalaVersion := "2.13.4"
```
Comment (kubukoz) [inline src/main/g8/build.sbt:2]: Would you mind bumping to 2.13.5?

**Review by kubukoz — CHANGES_REQUESTED** [review]

File: src/main/g8/build.sbt (hunk 2 above)
Comment (TonioGela) [reply src/main/g8/build.sbt]: Done

File: src/main/g8/src/main/scala/$package$/$name__Camel$.scala (hunk 1 above)
Comment (TonioGela) [reply src/main/g8/src/main/scala/$package$/$name__Camel$.scala]: Seems legit

**Comment by kubukoz** [comment]
Thanks

**Pull request merged**